roles/ceph-defaults/defaults/main.yml

```diff
@@ -77,7 +77,7 @@ debian_package_dependencies: []
 
 centos_package_dependencies:
   - epel-release
-  - libselinux-python
+  - "{{ 'libselinux-python' if ansible_distribution_major_version | int != 8 else 'python3-libselinux' }}"
```

Comment:
@guits If we only support CentOS 8 then we can use `python3-libselinux` only. wdyt ?